client/templates/meetingseries/meetingSeriesSearch.js

```diff
@@ -0,0 +1,19 @@
+import { Template } from 'meteor/templating';
+import { MeetingSeries } from '/imports/meetingseries';
+import { Session } from 'meteor/session';
+
+Template.meetingSeriesSearch.events({
+    'keyup .meetingSeriesSearchbar': function (event) {
+        const target = event.currentTarget;
+        const text = target.searchMeetingSeries.value;
+        Session.set('search-query', text);
+    }
+});
+
+Template.meetingSeriesSearch.searchResults = function () {
+    let wholeString = Session.get('search-query');
+    let split = wholeString.match(/\w+/g);
```

Comment:
This RegExp groups only for Characters [A-Za-z] - so it is not possible to search with other (non Western) languages and it is not possible to search e.g. for German Umlauts (öäü ß).
I will merge this PR and provide a Fix like so: 

    let split = wholeString.match(/[^\s]+/g);
    if (!split) {
        split = [];
    }

This splits on Whitespace and allows for search words consisting of all non-whitespace chars.